Simplenote/Classes/SPCardViewController.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+// MARK: - SPCardViewController
+//
+final class SPCardViewController: UIViewController {
+    private let viewController: UIViewController
+    private let containerView = UIView()
+
+    init(viewController: UIViewController) {
+        self.viewController = viewController
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .clear
+
+        addShadowView()
+        setupContainerView()
+        addChildViewController()
+    }
+}
+
+// MARK: - Private Methods
+//
+private extension SPCardViewController {
+    func addShadowView() {
+        let shadowView = SPShadowView(cornerRadius: Constants.cornerRadius,
+                                      roundedCorners: [.topLeft, .topRight])
+        view.addFillingSubview(shadowView)
+    }
+
+    func setupContainerView() {
+        view.addFillingSubview(containerView)
+
+        containerView.backgroundColor = UIColor.simplenoteBackgroundColor.withAlphaComponent(0.97)
```

Comment:
Nipticky: Maybe `0.97` > constant?

Reply:
Detail: You may need to listen to `VSThemeManagerThemeDidChangeNotification` and refresh the `.backgroundColor`.

TL;DR: Simplenote has supported, for a few years, dark mode. As per iOS 13 you don't need to do anything, for legacy support would require a manual refresh.

Reply:
Thanks for the info about `VSThemeManagerThemeDidChangeNotification `. I wasn't aware of it, will add support for it!